src/NLog.Targets.Syslog/MessageSend/Tcp.cs

```diff
@@ -104,7 +104,11 @@ private Stream SslDecorate(TcpClient tcpClient)
 
             // Do not dispose TcpClient inner stream when disposing SslStream (TcpClient disposes it)
             var sslStream = new SslStream(tcpStream, true);
+#if NETSTANDARD1_3 || NETSTANDARD1_5
+            sslStream.AuthenticateAsClientAsync(Server, retrieveClientCertificates(), SslProtocols.Tls12, false).Wait();
+#else
             sslStream.AuthenticateAsClient(Server, retrieveClientCertificates(), SslProtocols.Tls12, false);
+#endif
 
```

Comment:
Calling a blocking Wait on an async method is useless: please keep the old code (second statement)

Reply:
Fixed